pom.xml

```diff
@@ -100,7 +100,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>${version.jackson}.1</version>
+        <version>${version.jackson}.3</version>
```

Comment:
I have NO idea why the version is written like this. How about we just move the .3 up to the version.jackson property.

Reply:
I'll make the change after merge.

Reply:
It's because the jackson-databind artifact was released with some CVE and other fixes, separately from the rest of jackson, and the developer added an extra version component. You can't update `${version.jackson}` with the `.3`, because the other jackson libs don't have a version like that... only jackson-databind... and it should be temporary. You could create a separate `${version.jackson-databind}` property.